server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSLexer.g4

```diff
@@ -112,6 +112,7 @@ CHECK : C H E C K;
 CHANGEAGENT: C H A N G E A G E N T;
 CHANGEAGREL: C H A N G E A G R E L;
 CHANGEUSRID: C H A N G E U S R I D;
+CHILD: C H I L D;
```

Comment:
Please add token to keyword suggestions which can be found under LanguageKeywords.txt